packages/fxa-settings/src/pages/ResetPasswordRedesign/AccountRecoveryConfirmKeyOtp/container.tsx

```diff
@@ -0,0 +1,139 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import { RouteComponentProps, useLocation, useNavigate } from '@reach/router';
+import base32Decode from 'base32-decode';
+
+import { decryptRecoveryKeyData } from 'fxa-auth-client/lib/recoveryKey';
+import {
+  AuthUiErrors,
+  getLocalizedErrorMessage,
+} from '../../../lib/auth-errors/auth-errors';
+import { useAccount } from '../../../models';
+import { useFtlMsgResolver } from '../../../models/hooks';
+
+import {
+  AccountRecoveryConfirmKeyOtpContainerProps,
+  AccountRecoveryKeyOtpLocationState,
+} from './interfaces';
+
+import AccountRecoveryConfirmKeyOtp from '.';
+
+const AccountRecoveryConfirmKeyOtpContainer = ({
+  serviceName,
+}: AccountRecoveryConfirmKeyOtpContainerProps & RouteComponentProps) => {
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const account = useAccount();
```

Comment:
Can we avoid using `useAccount`? When used, it tries to fetch all of the account data that we use for Settings. OK in a follow up if that blows things up.

Reply:
I've filed a follow-up: `FXA-9672`